spec/support/spree.rb

```diff
@@ -6,5 +6,5 @@
 
 RSpec.configure do |config|
   config.include Spree::TestingSupport::UrlHelpers
-  config.include Spree::TestingSupport::ControllerRequests
-end
\ No newline at end of file
+  config.include Spree::TestingSupport::ControllerRequests, type: :controller
```

Comment:
No, that means we dont have to:

`RSpec.describe Thing, type: controller`

This means "Only include this for tests with type == :controller"